### ISSUE TITLE
[BUG] [Linter] T002 false positive

### ISSUE THREAD

**Issue by thekaveman** [Riverside-Healthcare/djLint#399]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Docker `python:3.10` image
 - Python Version: `Python 3.10.7`
 - djLint Version: `djlint, version 1.16.0`
 - template language: `django`

## Issue

On our most recent `pre-commit.ci` run, which updated from `1.14.0` to `1.16.0`, we had a new instance of `T002` pop up on old code that hasn't changed in a while.

* The [PR updating `djlint`](https://github.com/cal-itp/benefits/pull/923) is here
* The [pre-commit.ci run](https://results.pre-commit.ci/run/github/285501392/1663614943.HobZjazoQXWRBnkTsFdutQ) is here

The [template in question](https://github.com/cal-itp/benefits/blob/dev/benefits/core/templates/core/includes/form.html) looks like this:

```html
{% if form.action_url %}

  {% url form.action_url as form_action %}

  <form action="{{ form_action }}" method="{{ form.method | default:"post" | upper }}" role="form">

  ...
```

And the `djlint` error message is:

```console
benefits/core/templates/core/includes/form.html

T002 2:0 Double quotes should be used in tags. {% if form.action_ur

Linted 14 files, found 1 error.

100% 14/14 [00:00<?, ?it/s]        

Linted 14 files, found 0 errors.
```

## How To Reproduce

1. `git clone https://github.com/cal-itp/benefits.git`
2. `cd benefits && git checkout pre-commit-ci-update-config`
3. `pre-commit install --install-hooks`
4. `pre-commit run djlint-django --all`

## Contents of .djlintrc/pyproject.toml [tool.djlint]

```toml
[tool.djlint]
ignore = "H006,H017,H021,H025,H031"
indent = 2
max_attribute_length = 127
max_line_length = 127
profile = "django"
preserve_blank_lines = true
use_gitignore = true
```

**Comment by christopherpickering** [comment]
Thanks for reporting this!
